docs/react-v9/contributing/conducting-meetings-style-guide.md

```diff
@@ -0,0 +1,61 @@
+# Conducting Meetings Style Guide
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31758&commitId=4b4c27c16f70d321b4c19ea7cde92e6be306412d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|4| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Added|
|Avatar Converged.basic - RTL.normal.chromium.png|0| Removed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>Card Converged - Interactive</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|0| Removed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>